awscli/autoprompt/prompttoolkit.py

```diff
@@ -107,9 +110,15 @@ def _create_containers(self):
         return input_buffer_container, doc_window, output_window
 
     def create_application(self):
+        # There are four different levels of color depths available in python-prompt-toolkit, 
+        # which can be used to configure different color schemes in aws cli. You can read more
+        # about it in the documentation linked below:
+        # https://python-prompt-toolkit.readthedocs.io/en/master/pages/advanced_topics/styling.html?highlight=classes#color-depths
+        cliColorDepth = {"black_and_white": ColorDepth.MONOCHROME, "ansi_colors": ColorDepth.ANSI_COLORS_ONLY,"256_colors": ColorDepth.DEFAULT, "true_colors": ColorDepth.TRUE_COLOR}
```

Comment:
this code doesn't match the convention of the project, camelCase in python? should be changed to cli_color_depth. same for the depthLevel

Reply:
I'm interested in this feature too but they won't approve this convention

Reply:
You're welcome @Damans227 
I really hope this PR will be approved as it is very useful for me and many users too

Reply:
I've made the suggested changes. Let's hope for the best.